packages/expo-camera/ios/ExpoCamera.swift

```diff
@@ -0,0 +1,982 @@
+import UIKit
+import ExpoModulesCore
+import CoreMotion
+
+// swiftlint:disable:next type_body_length
+public class ExpoCamera: ExpoView, EXCameraInterface, EXAppLifecycleListener,
+  AVCaptureFileOutputRecordingDelegate, AVCapturePhotoCaptureDelegate {
+  public var session = AVCaptureSession()
+  public var sessionQueue = DispatchQueue(label: "captureSessionQueue")
+  private var motionManager = CMMotionManager()
+  private var physicalOrientation: UIDeviceOrientation = .unknown
+
+  // MARK: Legacy Modules
+  private var faceDetector: EXFaceDetectorManagerInterface?
+  private var lifecycleManager: EXAppLifecycleService?
+  private var barCodeScanner: EXBarCodeScannerInterface?
+  private var fileSystem: EXFileSystemInterface?
+  private var permissionsManager: EXPermissionsInterface?
+
+  // MARK: Properties
```

Comment:
Use markers with `-`, then Xcode shows an additional line separator in the method navigator (make it easier to navigate).
```suggestion
  // MARK: - Properties
```